src/components/WebexMemberRoster/WebexMemberRoster.scss

```
@@ -0,0 +1,7 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-roster-container {
```

For consistency with other components, the top-level CSS class name should match the component name
```suggestion
.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-member-roster {
```